mmdeploy/codebase/mmpose/deploy/pose_detection_model.py

```diff
@@ -159,6 +159,94 @@ def pack_result(self,
         return data_samples
 
 
+@__BACKEND_MODEL.register_module('yolox-pose_end2end')
+class YoloXPoseEnd2EndModel(End2EndModel):
+    """End to end model for inference of pose detection.
+
+    Args:
+        backend (Backend): The backend enum, specifying backend type.
+        backend_files (Sequence[str]): Paths to all required backend files(e.g.
+            '.onnx' for ONNX Runtime, '.param' and '.bin' for ncnn).
+        device (str): A string represents device type.
+        deploy_cfg (str | mmengine.Config): Deployment config file or loaded
+            Config object.
+        model_cfg (str | mmengine.Config): Model config file or loaded Config
+            object.
+        data_preprocessor (dict | nn.Module | None): Input data pre-
+                processor. Default is ``None``.
+    """
+
+    def __init__(self,
```

Comment:
if this method is the same as `End2EndModel`, then, could remove this part.

Reply:
yolox-pose do not need this `build_head` func:
https://github.com/open-mmlab/mmdeploy/blob/90f17f2e62ef3079dfcdb21c364892a43a2ff210/mmdeploy/codebase/mmpose/deploy/pose_detection_model.py#L57